src/compute/src/render/reduce.rs

```diff
@@ -387,15 +412,15 @@ where
                     // by the collation. If we cannot, we log an error and produce no output for
                     // this key.
                     let message = "Mismatched aggregates for key in ReduceCollation";
-                    warn!(
-                        ?key,
-                        debug_name,
-                        n_aggregates_requested = input.len(),
-                        n_distinct_aggregate_types,
-                        "[customer-data] {message}"
+                    error_logger.log(
+                        message,
+                        &format!(
+                            "[customer-data] key={key:?}, debug_name={debug_name}, \
+                             n_aggregates_requested={requested}, \
+                             n_distinct_aggregate_types={n_distinct_aggregate_types}",
+                            requested = input.len(),
+                        ),
```

Comment:
This fragment, and others similar to it, seem to change the previous code to omit `message` from what is written to the `warn!` output. Is this intentional? I would find it a bit easier if `message` were included in the warning as well, so that it is easier to match up in the output logs; this is just personal preference, of course.

Reply:
I think the message in the warning adds some value to sort out where these breadcrumbs are coming from without having to refer directly to the code. We may also be getting some of these outputs from multiple operators if they are all seeing invalid accumulations at once.

Reply:
The `ErrorLogger` now always logs the `message` in the WARN log as well, to enable this matching. This is also true for instances (mostly calls to `soft_panic_or_log`) where this previously was not the case (i.e. the ERROR message was different from the WARN message).